packages/notifi-dapp-example/src/app/notifi/dashboard/page.tsx

```diff
@@ -41,17 +44,48 @@ export default function NotifiDashboard() {
 
   return (
     <div className="min-h-screen flex items-start flex-row">
+      {/* desktop sidebar */}
       <DashboardSideBar
         account={account}
         cardView={cardView}
         setCardView={setCardView}
+        className="md:block hidden"
       />
-      <div className=" flex flex-col grow h-screen ">
+      {/* mobile sidebar */}
+      {isSideNavOpen ? (
+        <DashboardSideBar
+          account={account}
+          cardView={cardView}
+          setCardView={setCardView}
+          setIsSideNavOpen={setIsSideNavOpen}
+          className="md:hidden block bg-gradient-injective"
+        />
+      ) : null}
+      <div className="flex flex-col grow h-screen">
+        <div className="md:hidden w-screen flex justify-center">
+          <Icon
+            id="btn-nav"
+            className="top-6 left-4 cursor-pointer fixed"
+            onClick={() => setIsSideNavOpen(true)}
+          />
+          <Image
+            className="mt-3"
+            src="/logos/injective.png"
+            width={115}
+            height={24}
+            alt="Injective"
+            unoptimized={true}
+          />
+        </div>
         {unverifiedDestinations.length > 0 && cardView === 'history' ? (
           <VerifyBanner setCardView={setCardView} />
         ) : null}
         {/* IMPORTANT: Do not remove `min-h-0` , This is to fix the inner card height */}
-        <div className="flex flex-col grow bg-white rounded-3xl mb-10 mt-3 mr-10 min-h-0 shadow-card">
+        <div
+          className={`flex flex-col grow bg-white rounded-3xl md:mb-10 mt-3 md:mr-10 ${
+            cardView === 'alertSubscription' ? '' : 'min-h-0'
```

Comment:
Not sure if this is the expected behavior? (below)

https://github.com/notifi-network/notifi-sdk-ts/assets/127958634/0320cac3-87af-44a2-a569-c8ae086f1abf

Reply:
let's leave it like this for now, I'll ask Sam's opinion on tomorrow's final UAT